metasv/extract_pairs.py

```diff
@@ -129,10 +133,10 @@ def extract_read_pairs(bamname, region, prefix, extract_fns, pad=0, max_read_pai
     else:
         logger.info("Too many reads encountered for %s. Skipping read extraction. (%d >%d)"%(region, len(aln_dict),max_read_pairs))
 
-    bam.close()
-
     ends = [(open("%s_%s_1.fq" % (prefix, name), "w"), open("%s_%s_2.fq" % (prefix, name), "w")) for name in
             extract_fn_names]
+
+    chr_tid = bam_handles[0].gettid(chr_name) if bam_handles else -1
```

Comment:
Why is this from the first BAM file? Do you mean by this *any* BAM file, and why?